rest/src/test/scala/io/udash/rest/raw/ServerImplApiTest.scala

```diff
@@ -13,12 +14,52 @@ class ServerImplApiTest extends AnyFunSuite with ScalaFutures {
 
   private val apiImpl = new SomeServerApiImpl
   private val serverHandle = SomeServerApiImpl.asHandleRequest(apiImpl)
+  private val serverHandleStreaming = SomeServerApiImpl.asHandleRequestWithStreaming(apiImpl)
```

Comment:
Add a testcase for using streaming method with non-streaming server implementation